KicadModTree/nodes/base/Pad.py

```diff
@@ -47,7 +51,7 @@ def __init__(self, **kwargs):
         self._initLayers(**kwargs)
 
     def _initNumber(self, **kwargs):
-        self.number = kwargs.get('number')
+        self.number = kwargs.get('number', '""')  # default to an un-numbered pad
```

Comment:
@pointhi if you think this is the better idea, where should this code go?

I like to default to an "un-numbered" pad for e.g. mechanical mounting pads